src/js/preshuffle.s

```diff
@@ -781,15 +781,19 @@ MaybeSetCheckpointActual:
   .byte $20,$20,$1f     ;  _ _ |
 
 .org $bb1b
-  .byte $20             ;  _
-.org $bb1f ; clear out the experience count thats there initially
-.repeat 16
+  .byte $82             ; Ey
+; clear out the experience count thats there initially
+; and make room for something like enemy name
+.repeat 11
   .byte $20 ;  _
 .endrep
+  ; hold spots here for the enemy hp amount
+  .byte $20,$20,$20,$9d ; _ _ _ /
+  .byte $20,$20,$20,$20 ; _ _ _ _
```

Comment:
Thoughts about _not_ displaying enemy's MaxHP?  It would simplify the code quite a bit and removes some extra clutter from the HUD.  If you attention to the first hit, you can see (more or less) what it was.